apps/app/components/AppMenu/index.tsx

```diff
@@ -137,6 +137,8 @@ function AppMenu({ hidePrimaryMenu, collapsed }: any) {
                 <SubMenu icon={<CalendarOutlined />} title="Eventos">
                   <Item key="/events">Listar eventos</Item>
                   <Item key="/admin/event">Criar evento</Item>
+                  <Item key="/admin/lectures">Listar sessões</Item>
+                  <Item key="/admin/lectures/new">Criar sessão</Item>
```

Comment:
I think this would be prettier on a `SubMenu` called "Sessões". Following what's on a mentor a view, for example.